tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +53,8 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert result == False
```

Comment:
Excellent assert! It would also be a good idea to check to make sure the inventory length is still the same and nothing accidentally got removed.